app/src/main/java/chat/rocket/android/RocketChatApplication.java

```diff
@@ -23,6 +23,7 @@
   public void onCreate() {
     if (BuildConfig.DEBUG) {
```

Comment:
Thanks @luciofm ! That blog post is so great (I found it last week while browsing the `stetho_realm` repository :)) We can approach it in other situations too!

Of course they should have a NoOp like leakcanary does! :P

Do you think we need to change our `circle.yml` or create a `src/debug` folder?